app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -218,4 +224,37 @@ class ConversationViewModel @Inject constructor(
             )
         }
     }
+
+    private suspend fun fromMessageModelToMessageContent(message: com.wire.kalium.logic.data.message.Message): MessageContent =
+        when (val content = message.content) {
+            is Asset -> {
+                val assetId = content.value.remoteData.assetId
+                val assetToken = content.value.remoteData.assetToken
+                val privateKey = content.value.remoteData.otrKey
+                val (imgWidth, imgHeight) = when (val metadata = content.value.metadata) {
+                    is Image -> metadata.width to metadata.height
+                    else -> 0 to 0
+                }
+                val decodedImgDataResult = getMessageAsset(
+                    assetKey = assetId,
+                    assetToken = assetToken,
+                    conversationId = message.conversationId,
+                    messageId = message.id,
+                    encryptionKey = privateKey
+                ).run {
+                    when (this) {
+                        is PrivateAssetResult.Success -> decodedAsset
+                        else -> ByteArray(16)
```

Comment:
I saw this ByteArray(16), why are we returning it actually ? and why not ByteArray(15) ? :P just confusing why it is here, maybe a comment would do with a explanation

Reply:
I return for now a default byte array in case there was an error downloading the asset data. Later on, a default placeholder will be shown instead inside the `MessageImage` object since the call to `rawImgData.toBitmap()` will fail on an empty byte array. 
About the 16, it's just a convention for byte array initialisation. 
I will add a comment for now.

Reply:
if you just add a comment what you wrote above, it would be aweasome, because it explains it all nicely